# ロボットバージョンの名称SET
string name
---

